File: KuantumKaos/csharp/kuantumKaos/KuantumNesnesi.cs
using System;

namespace kuantumKaos
{
    public abstract class KuantumNesnesi
    {
        public string ID { get; set; } = "";

        private double stabilite;
        public double Stabilite
        {
            get { return stabilite; }
            set
            {
                if (value < 0)
                    stabilite = 0;
                else if (value > 100)
                    stabilite = 100;
                else
                    stabilite = value;
            }
        }

        public int TehlikeSeviyesi { get; set; }

        public abstract void AnalizEt();

        public string DurumBilgisi()
        {
            return $"ID: {ID} | Stabilite: {Stabilite}";
        }
    }
}

File: KuantumKaos/java/kuantumKaosJava/src/KuantumNesnesi.java
public abstract class KuantumNesnesi {
    protected String id;
    protected int stabilite;

    public KuantumNesnesi() {
        this.id = java.util.UUID.randomUUID().toString();
        this.stabilite = 100;
    }

    public String getId() {
        return id;
    }

    public int getStabilite() {
        return stabilite;
    }

    public void setStabilite(int stabilite) {
        this.stabilite = stabilite;
    }

    public abstract void analizEt();
}
